data/MCC-unsupported-zones

```diff
@@ -94,8 +94,9 @@ MQ
 # the locations formerly included in Netherlands Antilles.
 ^362.*AN
 
-# Belgium shares 2 networks with Luxembourg 270 77 and 270 99, even though most
+# Belgium shares 2 networks with Luxembourg 270 07, 270 77 and 270 99, even though most
```

Comment:
That's 3 now?

Reply:
This was actually incorrect, fixed to correct unsupported zone.